themes/classic/templates/layouts/layout-both-columns.tpl

```diff
@@ -61,9 +61,11 @@
 
           {block name="left_column"}
             <div id="left-column" class="col-xs-12 col-sm-4 col-md-3">
+              {block name="column_contact"}{/block}
+
               {if $page.page_name == 'product'}
                 {hook h='displayLeftColumnProduct'}
-              {else}
+              {else if $page.page_name != 'contact'}
```

Comment:
And it feels weird to have special use case just for the contact page, since this is a generic global layout
I don't doubt something needs to be fixed but it should be managed in the contact page only not in the général layouts Or we could end up with multiple `elseif` for each use case or pages..

Reply:
Look like 66 :roll_eyes:

Reply:
True... but at least the hook is related to product But in this case I dont really understand the logic of adding a new block and ignoring the generic hook When the initial hook worked no?

Reply:
I agree with you, we need to handle these block populate in the contact tpl, but we need a way to detect which layout is used in order to populate the right block.. If there is a way I would love to know it